projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -83,9 +95,16 @@
 	flex-wrap: wrap;
 }
 
-.action-link-button:hover {
-	color: var(--jp-black);
-	text-decoration-thickness: var(--jp-underline-thickness);
+.action-link-button {
+	text-decoration: underline;
+	font-size: 13px;
+    font-weight: 500;
```

Comment:
Can you highlight this usage in an issue, please? I'm trying to remove all `font-size` and `font-weight` alongside the application to keep consistency, use only components and enforce the proposed variants from design: PbJqbW-8t-p2 ...

Reply:
Btw, this `13px` font-size doesn't match any of our typography styles.

Reply:
I've removed the font size.